python/perspective/perspective/tests/core/test_widget_pandas.py

```diff
@@ -51,6 +51,10 @@ def superstore(count=50):
 class TestWidgetPandas:
 
     def test_widget_load_table_df(self):
+        '''Flaky on Python 2 - sometimes unicode encoding errors occur.
```

Comment:
My superstore can generate non English character names, give it a bigger size to guarantee the error

Reply:
fixed a few commits ago - removed outdated comment